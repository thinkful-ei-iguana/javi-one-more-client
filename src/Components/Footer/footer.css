footer{
    
    background-color: #282c34;
    color: white;
    border-top: 1px solid black;
    
    height: 2.5rem;
}