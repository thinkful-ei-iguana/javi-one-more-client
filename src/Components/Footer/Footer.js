import React from 'react'
import '../Footer/footer.css'

export default function Footer(){
        return (
            <div>
                <footer>
                    copyright 2019, one more
                </footer>
                
            </div>
        )
    }

