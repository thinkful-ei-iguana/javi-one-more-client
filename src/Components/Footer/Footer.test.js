import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom'
import Footer from '../Footer/Footer';
import renderer from 'react-test-renderer'


describe('add workout' , () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<BrowserRouter><Footer /></BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<BrowserRouter><Footer /></BrowserRouter>)
      .toJSON()
    expect(tree).toMatchSnapshot();
  })
})