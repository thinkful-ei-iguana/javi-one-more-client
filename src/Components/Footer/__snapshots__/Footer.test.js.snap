// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add workout renders the UI as expected 1`] = `
<div>
  <footer>
    copyright 2019, one more
  </footer>
</div>
`;
